app/routes/public/sessions.js

```diff
@@ -130,8 +130,16 @@ export default class SessionsRoute extends Route {
     }
 
     return {
-      event   : eventDetails,
-      session : await this.infinity.model('sessions', {
+      event       : eventDetails,
+      dateSession : await this.infinity.model('sessions', {
+        sort         : params.sort || 'starts-at',
+        perPage      : 6,
+        startingPage : 1,
+        perPageParam : 'page[size]',
+        pageParam    : 'page[number]',
+        store        : eventDetails
+      }),
```

Comment:
Why?

Reply:
when the session filter changes then the date also changes related to that filter. And on that date filter only that is gonna available.

Reply:
Sounds like a workaround

Reply:
no, it's not a work around.

Reply:
Once the dates are set, make it so they don't change again

Reply:
I have been trying it till now, but couldn't find another way.